src/calendar/header/style.js

```diff
@@ -26,7 +26,19 @@ export default function(theme={}) {
       ...appStyle.arrowStyle
     },
     arrowImage: {
-      tintColor: appStyle.arrowColor
+      ...Platform.select({
```

Comment:
`Platform` is not defined - you have a lint error

Reply:
I have simplified the logic by removing Platform.